openapi/openapi.yaml

```
@@ -470,7 +470,44 @@ components:
         - "Fixed Asset"
         - "Lump Sum"
         - "ITT, Vehicles and Equipment"
-      descirption: The type of Capital Project.
+      description: The type of Capital Project.
+    CapitalProject:
+      type: object
+      properties:
+        id:
+          type: string
+          description: A string of variable length, composite of managing code and project ID
```

There is no single field that represents the whole of the composite key. The composite key exists as a combination of the "id" and "managingCode". I recognize the ambiguity of calling it 'id'; it makes it read like a thing that should be unique. Unfortunately, I don't think we're going find a perfect name. We just need to remember that `id` is not the composite key. It is one of the two fields that compromises the composite key. 

```suggestion
          description: The id for the project, which combines with the managing code to make a unique id
```

---

Wait so the example of this is `HWPEDSF5`-- is that the ID that gets combined with a separate managing code? and `HWPEDSF` big is just part of the project ID?

---

Ok yes because managing codes are numbers. Okay cool